docs/Users/Tutorials/Linter_Bears.rst

```diff
@@ -14,289 +14,286 @@ need to implement it on your own. Don't worry, it's easy!
 Since it is a python package we can go ahead and install it with
 
 ::
 
-    $ pip install html-linter
+    $ pip3 install pylint
 
 Writing the Bear
 ----------------
 
-Since we are going to use the Lint class we should go ahead and
-import it together with LocalBear (all bears that handle only one file
-inherit from LocalBear). Also we will go ahead and write the class
-head. It should inherit both LocalBear and Lint.
+To write a linter bear, we need to create a class that interfaces with our
+linter-bear infrastructure, that is provided via the linter decorator.
 
 ::
 
-    from coalib.bearlib.abstractions.Lint import Lint
-    from coalib.bears.LocalBear import LocalBear
+    from coalib.bearlib.abstractions.Linter import linter
 
-    class HTMLTutorialBear(LocalBear, Lint):
+    @linter(executable='pylint')
+    class PylintTutorialBear:
+        pass
 
-To make our bear use a linter we will have to overwrite some of the
-predefined values of the Lint class. Some of the most important are
-``executable`` and ``output_regex``.
+As you can see ``pylint`` is already provided as an executable name which gets
+invoked on the files you are going to lint. This is a mandatory argument for
+the decorator.
 
-We use ``executable`` to specify the linter executable. In our case it would
-be
+The linter class is only capable of processing one file at a time, for this
+purpose ``pylint`` or the external tool needs to be invoked every time with the
+appropriate parameters. This is done inside ``create_arguments``,
 
 ::
 
-    executable = 'html_lint.py'
+    @linter(executable='pylint')
+    class PylintTutorialBear:
+        @staticmethod
+        def create_arguments(filename, file, config_file):
+            pass
 
-The ``output_regex`` is used to group parts of the output (such as ``lines``,
-``columns``, ``severity`` and ``message``) so it can be used by the Lint
-class to yield Results (more on communicating with the user
-:doc:`Writing Bears <Writing_Bears>`).
+``create_arguments`` accepts three parameters:
 
-In order to figure out the ``output_regex`` we have to first see how the
-linter output looks. I will use this file as ``sample.html``
+- ``filename``: The absolute path to the file that gets processed.
+- ``file``: The contents of the file to process, given as a list of lines
+  (including the return character).
+- ``config_file``: The absolute path to a config file to use. If no config file
+  is used, this parameter is ``None``. More on that later.
 
-::
+You can use these parameters to construct the command line arguments. The
+linter expects from you to return an argument sequence here. A tuple is
+preferred. We will do this soon for ``PylintTutorialBear``.
 
-    <html>
-      <body>
-        <h1>Hello, world!</h1>
-      </body>
-    </html>
+So which are the exact command line arguments we need to provide? It depends on
+the output format of the linter. The linter decorator is capable of handling
+different output formats:
```

Comment:
we may implement later more formats, that's why I wanted to be general^^ Shall I change anyway?